addons/Fractions/src/presenter.js

```diff
@@ -1481,5 +1856,16 @@ function AddonFractions_create(){
         return elements;
     };
 
+    function getIndexesInOrder(numberOfElements) {

```

Comment:
Nie ma ograniczenia do 16. Można mieć nawet większe liczby np. 32. 
![image](https://github.com/user-attachments/assets/564cc232-0786-4000-8385-c1bffbcc5db5)
W zadaniu nie widziałem informacji, że tylko do 16 powinniśmy wspierać.
Najlepszym rozwiązaniem byłoby znalezienie algorytmu pasującego do indeksacji. Jeżeli niemożliwe to ustalenie z LEA do ilu wspieramy i w dokumentacji powinna wtedy znaleźć się o tym informacja